Common/include/option_structure.hpp

```diff
@@ -1162,17 +1164,24 @@ enum ENUM_PARAM {
   AIRFOIL = 17,		           /*!< \brief Airfoil definition as design variables. */
   SURFACE_FILE = 18,		     /*!< Nodal coordinates set using a surface file. */
   CUSTOM = 19,                /*!< 'CUSTOM' for use in external python analysis. */
-  CST = 20                /*!< \brief CST method with Kulfan parameters for airfoil deformation. */
+  CST = 20,                /*!< \brief CST method with Kulfan parameters for airfoil deformation. */
+  NO_DEFORMATION = 21,		       /*!< \brief No Deformation. */
+  HTP_INCIDENCE = 22,			         /*!< \brief Incidence of the HTP. */
+  ANGLE_OF_ATTACK = 101,	   /*!< \brief Angle of attack for airfoils. */
+  FFD_ANGLE_OF_ATTACK = 102	 /*!< \brief Angle of attack for FFD problem. */
 };
 static const map<string, ENUM_PARAM> Param_Map = CCreateMap<string, ENUM_PARAM>
 ("FFD_SETTING", FFD_SETTING)
 ("FFD_CONTROL_POINT_2D", FFD_CONTROL_POINT_2D)
+("FFD_ANGLE_OF_ATTACK", FFD_ANGLE_OF_ATTACK)
 ("FFD_CAMBER_2D", FFD_CAMBER_2D)
 ("FFD_THICKNESS_2D", FFD_THICKNESS_2D)
 ("HICKS_HENNE", HICKS_HENNE)
+("ANGLE_OF_ATTACK", ANGLE_OF_ATTACK)
```

Comment:
Now SU2 screen output provides the value of the sensitivity with respect to the AoA. This is more useful than with respect to Mach. Remember that there are two ways to design using AoA (directly or indirectly), you can add AoA as a design variable (#101) or you can design at a cte. CL (and the angle of attacks is automatically adjusted).